tests/test_bitinformation_pipeline.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+import xarray as xr
+
+import bitinformation_pipeline as bp
+
+
+def test_full():
+    """Test bitinformation_pipeline end to end."""
+    label = "rasm"
+    ds = xr.tutorial.load_dataset(label)
+    # bitinformation_pipeline
+    bitinfo = bp.get_bitinformation(ds, dim=3)
+    keepbits = bp.get_keepbits(ds, bitinfo)
+    ds_bitrounded = bp.xr_bitround(ds, keepbits)
+    # save
+    ds.to_netcdf(f"{label}.nc")
+    ds.to_compressed_netcdf(f"{label}_compressed.nc")
+    ds_bitrounded.to_compressed_netcdf(f"{label}_bitrounded_compressed.nc")
```

Comment:
thats the end2end workflow and API. suggestions to modify? @observingClouds @milankl

Reply:
is `xr_bitround` the bitround from xarray as discussed [here](https://github.com/zarr-developers/numcodecs/issues/298)? For performance I'd suggest to use `round` from BitInformation. I've never actually compared the speed, but recalculation of constants is avoided reaching 16GB/s at which point it's memory-bound (and therefore likely an upper bound). Also gives us more flexibility in case we want to change something on the rounding front.

Reply:
Note that netcdf compression in 4.9 will have zstandard implemented next to zlib, so we should have an argument to choose between the two too (but zstd default) and compression level should also be flexible in case you want more speed vs more compression

Reply:
> how are you going to pass on arguments? In the future we may want to add further keyword arguments to `bitinformation` and it might be smart to build the pipeline such they are directly passed on for future compatibility.

`bp.get_bitinformation(ds, dim=3)` take all arguments: https://github.com/observingClouds/bitinformation_pipeline/blob/4f52bd0cff79b897a9f00f5e00b31d1fed807f0b/bitinformation_pipeline/bitinformation_pipeline.py#L84 I pass arguments `kwargs` https://github.com/observingClouds/bitinformation_pipeline/blob/4f52bd0cff79b897a9f00f5e00b31d1fed807f0b/bitinformation_pipeline/bitinformation_pipeline.py#L45 as strings to `jl.eval("get_bitinformation(X,{kwargs_string})")` - this is independent of the keyword but so far only works for float or bool and not for arrays as kwargs but with `masked_value=NaN` and data from `xarray` I think we do not need `mask`

Reply:
> is xr_bitround the bitround from xarray as discussed https://github.com/zarr-developers/numcodecs/issues/298?

exactly. We can and will check performance later

Reply:
> Note that netcdf compression in 4.9 will have zstandard implemented next to zlib, so we should have an argument to choose between the two too (but zstd default) and compression level should also be flexible in case you want more speed vs more compression

I think the function `to_compressed_netcdf` is a wrapper for a quick output, but in general users should probably just use `to_netcdf` and provide the encoding themselves, e.g. the compression algorithm. Or any user provided encoding to `to_compressed_netcdf` could overwrite its standard.

Reply:
Yep. I wanted to provide a working default. I had no idea how to compress initially